src/components/dropdown/README.md

```diff
@@ -308,6 +308,37 @@ split button its own variant via the `split-variant` prop.
 <!-- b-dropdown-split-variant.vue -->
 ```
 
+### Block level dropdowns
+
+By default dropdowns act like buttons and are displayed inline. To create block level dropdowns
+(that span to the full width of a parent) set the `block` prop. Both, regular and split button
+dropdowns are supported.
+
+```html
+<div>
+  <b-dropdown text="Block Level Dropdown" block variant="primary" class="m-2">
+    <b-dropdown-item href="#">Action</b-dropdown-item>
+    <b-dropdown-item href="#">Another action</b-dropdown-item>
+    <b-dropdown-item href="#">Something else here</b-dropdown-item>
+  </b-dropdown>
+
+  <b-dropdown
+    text="Block Level Split Dropdown"
+    block
+    split
+    split-variant="outline-primary"
+    variant="primary"
+    class="m-2"
+  >
+    <b-dropdown-item href="#">Action</b-dropdown-item>
+    <b-dropdown-item href="#">Another action</b-dropdown-item>
+    <b-dropdown-item href="#">Something else here...</b-dropdown-item>
+  </b-dropdown>
+</div>
+
+<!-- b-dropdown-block.vue -->
+```
+
```

Comment:
If users also want the menu to be the same width as the block level button, you could mention that they could add prop:

```html
<b-dropdown text="Block Level Dropdown" block variant="primary" class="m-2" menu-class="w-100">
```

This will make the menu the same width as the block level button. Useful mainly for dropdown or dropup menus

Reply:
@tmorehouse Or should we also add a `menuBlock` prop?

Reply:
I thought about that... not not sure how it would be/look for dropright/dropleft menus... unless we check for dropdown/dropup only... and not add it when dropleft/dropright

Reply:
@tmorehouse Maybe we should stick with the mention for now.

Reply:
OK... And maybe an example